include/LightGBM/boosting.h

```diff
@@ -131,10 +131,10 @@ class LIGHTGBM_EXPORT Boosting {
   * \param output Prediction result for this record
   * \param early_stop Early stopping instance. If nullptr, no early stopping is applied and all models are evaluated.
   */
-  virtual void PredictRaw(const double* features, double* output,
+  virtual void PredictRaw(double* features, double* output,
                           const PredictionEarlyStopInstance* early_stop) const = 0;
 
-  virtual void PredictRawByMap(const std::unordered_map<int, double>& features, double* output,
+  virtual void PredictRawByMap(std::unordered_map<int, double>& features, double* output,
```

Comment:
Is that possible to update feature values before calling prediction? like have a function like GBDT::PrepraeFeatureValues, and call before prediction.